### DISCUSSION TITLE
Możliwy koniec bazy IDCAC

### DISCUSSION THREAD

**Discussion by krystian3w** [MajkiIT/polish-ads-filter#21287] [Show and tell] [answered]
https://www.ghacks.net/2022/09/15/avast-acquires-i-dont-care-about-cookies-browser-extension/

https://www-ghacks-net.translate.goog/2022/09/15/avast-acquires-i-dont-care-about-cookies-browser-extension/?_x_tr_sl=en&_x_tr_tl=pl&_x_tr_hl=pl

**Comment by hawkeye116477** [comment]
Czyli zapewne dołożą swoje reklamy do rozszerzenia albo coś innego zepsują albo nic się nie zmieni. Choć niby kiboke pisze, że nadal będzie kontynuował prace.

**Reply by krystian3w** [reply]
Gorzej jak jakieś studenciaki będą uzupełniać zgłoszone luki jako pierwsze by licznik ~20 tysięcy zgloszeń zmniejszyć do 0 (chyba powyżej 12 thousands kibole ukrył dokładną liczbę "wstydu").

Oczywiście z pomiajniem moderowania czy filtr będzie psuć wszelkie pop-upy lub storna zamroziła ramki przed banem usług Google.

**Reply by hawkeye116477** [reply]
A nawet gdyby, to jak to mawiają, Internet nie znosi próżni, tzn. może ktoś zrobi jakiegoś forka czy coś.

Tak czy siak, w przypadku przeglądarki Waterfox, podobna panika była, a w zasadzie nie zmieniło się nic, ani gorsze ani specjalnie lepsze, tzn. nadal jest kiepsko, za bardzo trzymają się Mozilli w odróżnienie od Moonchilda, który jednak wydaje się iść bardziej odważną niezależną drogą.

**Reply by AroKol78** [reply, marked as answer]
co sądzicie o https://github.com/OhMyGuus/I-Dont-Care-About-Cookies

**Reply by hawkeye116477** [reply]
Fajnie, że jest na GitHubie. Chociaż na razie raczej nie ma co sądzić, bo prace niby są w toku i pewnie żadnych różnic póki co nie ma. Nadal też jest ten mało czytelny kod w https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/blob/master/src/data/rules.js.
